djangocms_references/templates/djangocms_references/unpublish_dependencies.html

```diff
@@ -0,0 +1,43 @@
+{% load i18n djangocms_references_tags %}
```

Comment:
Perhaps we can reuse the template used for "Show references" view. Split the table to a separate template and `{% include %}` it in both "Show references" and this template?

Reply:
@chaosk I'm not sure we can... Is it definitely ok for the Show references one to use the same "No related objects" logic?

Reply:
Actually hmmmm... now that I've refactored the function the way I have, we probably could... ok I'll change it.